CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.5 - 2018-11-20
```

Comment:
nit: updating `CHANGELOG` and `gemspec` is usually the last step, if we don't want to bump version every time (ie. 1 version == multi PRs)